src/toxicReleaseLayer.js

```diff
@@ -0,0 +1,133 @@
+require('jquery') ; 
+require('leaflet') ; 
+
+
+
+L.Icon.ToxicReleaseIcon = L.Icon.extend({
+    options: {
+      iconUrl: 'http://www.clker.com/cliparts/r/M/L/o/R/i/green-dot.svg',
+      iconSize:     [30, 20], 
+      iconAnchor:   [20 , 0], 
+      popupAnchor:  [-5, -5] 
+    }
+});
+
+L.icon.toxicReleaseIcon = function () {
+    return new L.Icon.ToxicReleaseIcon();
+};
+
+
+L.LayerGroup.ToxicReleaseLayer = L.LayerGroup.extend(
+
+    {
+        options: {
+            url: 'https://iaspub.epa.gov/enviro/efservice/tri_facility/pref_latitude/BEGINNING/41/rows/0:100/JSON',
```

Comment:
@jywarren ...am i using the correct url ? Kindly verify this .

Reply:
The AJAX request takes a lot of time for this url ⌛️ .

Reply:
41 in the url is the latitude , i guess i should replace it with the map center latitude . What do you think ?